schema/build-config-schema.js

```diff
@@ -436,11 +451,16 @@ rules.forEach(function forRule(rule) {
   }
   if (custom) {
     // @ts-ignore
-    scheme.type = [ "boolean", "object" ];
+    scheme.type = ["boolean", "object"];
     scheme.additionalProperties = false;
   }
-  rule.names.forEach(function forName(name) {
-    schema.properties[name] = scheme;
+  rule.names.forEach(function forName(name, index) {
+    if (index == 0)
```

Comment:
The change makes sense, but is hard to read in the GitHub diff due to linting issues - please fix those and I'll have a closer look.